backend/apps/api/src/routes/image.py

```
@@ -1,13 +1,125 @@
+            "url": url,
+            "created_at": img.created_at.isoformat(),
+            "user_id": img.user_id,
+            "user": user  # ✅ 업로더 정보 포함
+        })
+    
+    return {"images": result, "count": len(result)}
+
+@router.get("/images")
+async def list_my_images(
+    user_id: int = Depends(get_current_user),
+    db: Session = Depends(get_db),
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)